tests/foreman/api/test_contentview.py

```diff
@@ -60,8 +62,50 @@
 REPEAT = 3
 
 
+@fixture(scope='class', autouse=True)
+def module_org(request):
+    request.cls.module_org = entities.Organization().create()
```

Comment:
It's idiomatic for pytest fixtures to return, or yield, value, not assign it to something.

Look at e.g. https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_activationkey.py#L68 :
```python
@fixture(scope='module')
def module_org():
    return entities.Organization().create()

...

def test_positive_search_scoped(session, module_org):
    repo_id = create_sync_custom_repo(module_org.id)
```

Reply:
The things work differently when we have to class module-level fixtures in a class and especially when the class in inheriting unittest.TestCase.

I don't want to completely replace the class level separation of tests.

You can find more about the fixtures when used with unit test in pytest documentation.

Reply:
I see, you are right. They cover it in document on [UnitTest support](https://docs.pytest.org/en/latest/unittest.html), with explanation provided in Note at the bottom (easy to miss!).